theme-contract/src/main/java/com/orange/ouds/theme/tokens/semantic/OudsElevationSemanticTokens.kt

```diff
@@ -12,36 +12,14 @@
 
 package com.orange.ouds.theme.tokens.semantic
 
-import androidx.compose.runtime.Stable
 import androidx.compose.ui.unit.Dp
 import com.orange.ouds.tokens.global.raw.ElevationRawTokens
 
-data class OudsElevationTokens(
+data class OudsElevationSemanticTokens(
     val none: Dp = ElevationRawTokens.elevation0,
```

Comment:
As discussed, we can use `Float` instead of `Dp` because other tokens such as borders use `Float`.